Makefile

```diff
@@ -825,15 +863,15 @@ ICON_TOOL := node_modules/.bin/fileicon
 # STATUS_PORT ?= 30306
 # WAKUV2_PORT ?= 30307
 
+run-linux: export LD_LIBRARY_PATH := $(QT5_LIBDIR):$(STATUSGO_LIBDIR):$(STATUSKEYCARDGO_LIBDIR):$(LD_LIBRARY_PATH)
 run-linux: nim_status_client
 	echo -e "\033[92mRunning:\033[39m bin/nim_status_client"
-	LD_LIBRARY_PATH="$(QT5_LIBDIR)":"$(STATUSGO_LIBDIR)":"$(STATUSKEYCARDGO_LIBDIR):$(LD_LIBRARY_PATH)" \
-	./bin/nim_status_client $(ARGS)
+	nixGL ./bin/nim_status_client $(ARGS)
```

Comment:
It should still be imho possible to run the binary natively (like I'm doing now); I'm not personally interested in running in a sandbox with an outdated Qt version when my system is perfectly fine

Reply:
I'll make a separate comment what I'd want; seems there's a general misundestanding :)

Reply:
@caybro You have a different case. You build and run with same libraries.
We build with Nix libraries, running it with system libraries will cause certain issues.
The same issues, which we fix by bundling libraries into AppImage.

As I mentioned already switching 5.15.8 to 5.15.15 in out Nix setup will not be a big problem.

What is the benefit of you developing with new version btw, if we still deploy an app built with Qt 5.15.2 and bundled with it's libraries?

Reply:
@alexjba I read through couple of times, but could not get it, let's talk on Discord.

But once again, you cant just run `nim_status_client` build with Nix outside of Nix shell.
It will require solving several issues.